data/file-lists/zs-tools/config_linux.py

```
@@ -2360,21 +2360,28 @@ def _config_nics_with_enable_ha(nics):
-                with open(bond_sysfs_mode, 'r') as f:
-                    # Format: "balance-xor 2" or "active-backup 1"
-                    mode_str = f.read().strip().split()[0]
-                    bond_mode = mode_str
-                    logger.info('detected existing bond mode from sysfs: %s' % bond_mode)
+
+        # If not bond mode, ensure 'master' is set correctly
+        # _retry_find_expect_nics_by_mac may have set 'bond' or 'available_nics' for migration states
+        # but for non-bond scenarios, we need 'master'
+        if not is_bond_nic:
+            if 'master' not in nic_dict:
+                if 'available_nics' in nic_dict and nic_dict['available_nics']:
```

Ruff (0.14.8)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)